code/renderers/react/src/public-types.test.tsx

```diff
@@ -172,10 +172,24 @@ describe('Story args can be inferred', () => {
       </>
     );
 
+    // decorator is not using args
+    const thirdDecorator: Decorator<Args> = (Story) => (
+      <>
+        <Story />
+      </>
+    );
+
+    // decorator is not using args
+    const fourthDecorator: Decorator<StrictArgs> = (Story) => (
```

Comment:
Is this obsolete, or do you want to add it to the array in line 192?

Reply:
Oh, good catch!

Reply:
fixed